torch/_ops.py

```diff
@@ -788,11 +786,77 @@ def tags(self):
 # TorchBindOpOverload are those custom ops which have at least one overload's
 # schema consists of torch.ScriptObject (i.e. custom class) input.
 # TorchBindOpOverload will skip C++ dispatcher and purely dispatched in python
-# when its inputs contain FakeScriptObject.
+# when its inputs contain FakeScriptObject in a similar way as higher order ops.
 class TorchBindOpOverload(OpOverload):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._is_torchbind_op = True
+
+        self.non_fallthrough_keys = torch._C._dispatch_keyset_full()
+
+        _TORCH_BIND_OP_DEFAULT_FALLTHROUGH_DISPATCH_KEYS = [
```

Comment:
- If the implementation is a fallthrough (as is the case for ADInplaceOrView or BackendSelect), then we shouldn't need to do anything -- I would expect `_get_dispatch` to just handle the case?
- If the implementation is a fallback for PythonDispatcher, PythonTLSSnapshot, or Autograd, then we should use a fallthrough for those. This is not completely correct; ideally we would be able to have python versions of the fallbacks that we can call.

Reply:
Sorry, let me think about this some more.